packages/tracking/src/hooks/use_track.ts

```diff
@@ -25,16 +25,15 @@ declare const window: Window & {
 
 
 type TrackEventMap = {
```

Comment:
`create` event missing.. probably comes later with some additional tracking story in future...